content/en/logs/guide/getting-started-lwl.md

```diff
@@ -18,7 +18,7 @@ further_reading:
 
 ## Overview
 
-Datadog [Logging without Limits][1]™ provides flexibility by decoupling [log ingestion and indexing][2]. Cloud-based applications can generate logs at a rate of millions per minute. This type of high volume log indexing is helpful when monitoring every aspect of a service, but it can be inhibiting when troubleshooting during high priority issues or tracking bugs.
+Cloud-based applications can generate logs at a rate of millions per minute. But because your logs are not all and equally valuable at any moment, Datadog [Logging without Limits][1]™ provides flexibility by decoupling [log ingestion and indexing][2].
```

Comment:
```suggestion
Cloud-based applications can generate logs at a rate of millions per minute. But because your logs are not all and equally valuable at any moment, Datadog [Logging without Limits][1]™ provides flexibility by decoupling [log ingestion and indexing][11].
```

Reply:
Nope, we do want to link to the introductory LwL video here.